src/Text/TeXMath/Writers/Typst.hs

```diff
@@ -250,7 +250,11 @@ mkArray :: [[[Exp]]] -> Text
 mkArray rows =
   T.intercalate "; " $ map mkRow rows
  where
-   mkRow = T.intercalate ", " . map mkCell
+   mkRow = T.intercalate ", " . mkCells . map mkCell
+   mkCells cs =
+    if length cs <= 1 || head cs /= ""
+      then cs
+      else "#none" : tail cs
```

Comment:
I'd prefer to avoid using unsafe partial functions like `head`:
```hs
case cs of
  (' ':rest) -> "#none" : rest
  _ -> cs
```

Reply:
Thanks for the advice! I was looking for a better solution, and this looks very neat!